### PR TITLE
Fix/event detail view

### PR DESCRIPTION
**Pull request by DrewBradley**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Add venue data to the event query.
Pass event data to Event Preview from Dashboard AND Calendar.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Lines 19-21 of `src/components/calendar/Calendar.js`
Lines 12-16 of `src/components/calendar/event-utils.js`
Lines 7-23 of `src/components/event/Event.js`

*How should this be manually tested?*
Open application and click on an event. See if that is the right event.
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/calendar/event-utils.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ export function formatEvents(eventsArray) {
       title: event.name,
       start: event.startTime,
       end: event.endTime,
+      extendedProps: {
```
Comment (kristenmb) [inline src/components/calendar/event-utils.js:12]: This is definitely the cleaner way to do this in terms of passing the information into the event details

File: src/components/event/Event.js [hunk 2]
```diff
@@ -2,24 +2,42 @@ import React, {useState} from 'react'
 import Agenda from '../agenda/Agenda'
 import './Event.css';
 
-const Event = () => {
+const Event = (props) => {
   const [agenda, setAgenda] = useState([])
+  const [currentEvent] = useState(props.location.state.eventInfo)
+  // console.log(eventDate)
+  
+  const formatDate = (eventDate) => {
```
Comment (kristenmb) [inline src/components/event/Event.js:10]: Maybe we can move this to a utils file to clean up the component

**Review by kristenmb — APPROVED** [review]

**Review by kevinhartmann23 — APPROVED** [review]
Sweeeeet!

**Review by richardltyler — APPROVED** [review]
Looks good!

**Pull request merged**